openedx/core/djangoapps/verified_track_content/management/commands/swap_from_auto_track_cohort_pilot.py

```diff
@@ -170,6 +170,7 @@ def handle(self, *args, **options):
                     )
                     if (audit_partition_group_access
                             and audit_course_user_group_partition_group.group_id in audit_partition_group_access):
+                        print "Queueing XBlock at location: '%s' for Audit Content Group update " % item.location
```

Comment:
Any specific reason you chose `print` over the more standard `log.info()`? I can't claim to know exactly why you'd prefer one over the other, but a quick search shows [we do it that way in other commands](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/management/commands/reset_grades.py#L15)